espnet2/asr/frontend/adapter_utils/add_adapters.py

```diff
@@ -0,0 +1,91 @@
+import numpy as np
+
+from espnet2.asr.frontend.adapter_utils.adapters.adapter_transformer import (
+    AdapterTransformerSentenceEncoderLayer,
+)
+
+
+def add_adapters_wav2vec2(wav2vec2_model, adapter_down_dim, adapt_layers=None):
+    """
+    add adapters to wav2vec2 model.
+    * adapter_down_dim - down-projection dimension of adapter.
+    * adapt_layers - list of indices of layers to insert adapters. If `None`, adapters are inserted to every layer. (default=`None`).
+    """
+    if adapt_layers == []:
+        print(">> adapt_layers is an empy list. No adapters will be inserted.")
+        return
+    orig_param_num = count_params(wav2vec2_model)
+
+    # freeze all layers
+    for param in wav2vec2_model.parameters():
+        param.requires_grad = False
+
+    adapted_layers = []
```

Comment:
What is the difference between `adapted _layers` and `adapt_layers`

Reply:
yes , it seems redundant will get rid of it , and will remove print , I think its better to write into train.log